test/__snapshots__/step-function.test.ts.snap

```diff
@@ -6401,17 +6401,15 @@ exports[`call AWS.DynamoDB.GetItem, then Lambda and return LiteralExpr 1`] = `
       "ResultPath": "$.person",
       "Type": "Pass",
     },
-    "person = await $AWS.DynamoDB.GetItem({Table: personTable, Key: {id: {S: inp": {
+    "person = await personTable.get.attributes({id: {S: input.id}})": {
       "Next": "person",
       "Parameters": {
-        "Key": {
-          "id": {
-            "S.$": "$.input.id",
-          },
-        },
         "TableName": "__REPLACED_TOKEN",
+        "id": {
+          "S.$": "$.input.id",
+        },
```

Comment:
regression?

Reply:
Yeah I've changed the interface but haven't updated the implementation. Will do that today